pipeline/scraper/github_query.py

```diff
@@ -100,8 +104,8 @@ def run_query(today):
     monthly_search_str = get_monthly_search_str(today)
     while True:
         try:
-            result = run_query_once(MAX_NODES_PER_LOOP, monthly_search_str, last_node)
-            write_db(database, result)
+            result = run_query_once(MAX_NODES_PER_LOOP, monthly_search_str, last_node, language)
+            write_db(database, result) if language == "JavaScript" else write_db_python(database, result)
```

Comment:
I think this would be more readable if you rename `write_db` to indicate the language it supports, and restructure the `if` statement as follows:

```suggestion
            if language == "JavaScript":
                write_db_javascript(database, result)
            elif language == "Python":
                write_db_python(database, result)
```